packages/docs/docs/exporting-from-figma.md

```diff
@@ -0,0 +1,71 @@
+---
+id: exporting-from-figma
+title: Exporting designs from Figma to Remotion
+---
+
+Say you're collaborating as a Frontend Engineer with a design team, and you want a way to get a real-time representation of some of the design assets you have on Figma.
+
+You can export a design from Figma as an SVG file and import it as a React component in Remotion.
+
+## Open the Figma design
+
+The design we'll consider here is a suite of 3D graphics made by [Alzea Arafat](https://dribbble.com/alzea) for the design community. You can find them [here](<https://www.figma.com/file/VCoVolpndRmxkSd5AvctYZ/SALY---3D-Illustration-Pack-(Community)?node-id=7%3A4>). When you click the link to the design, click on the **"Library"** &mdash; in the sidebar &mdash; text to access all 3D graphics
+
+![Thumbnail](/static/img/export-figma/banner.png)
+
+Click on any of the vector graphics. Make sure you're selecting the 3D element alone. Don't select the whole container, the image below illustrates this.
+
+When you select the vector element, a border will be around it to indicate that it is currently in focus. Looking at the image below, you'll notice on the right, there's a "right sidebar" component with different tabs namely, **Design**, **Prototype**, **Inspect**
+
+![exporting as SVG](/static/img/export-figma/export-as-svg.png)
+
+Stay on the **Design** tab, then scroll down till you see the **Export** text, click on the plus sign next to it, select the **SVG** format from the file type dropdown menu, then click the export button
+
+## Using the file in Remotion.
+
+Now that you have exported the SVG file, it should be in your downloads folder. You can copy and paste the file into your remotion project or move the file(s) into the project to avoid duplicates.
+
+When you open the SVG file, it'll look like the image below.
+
+![svg file example](/static/img/export-figma/svg-file.png)
+
+You should take note of attributes like `xmlns:xlink` or `xlink:href` because TypeScript will throw an error due to how html attributes are used in React, these attributes have to be in `camelCase` form.
```

Comment:
Maybe we can link https://react-svgr.com/playground/ as the best way to convert SVG into a React component.

Reply:
Great! I'll check this out too.